### PR TITLE
Jetpack section: Add Backup and Scan cards to the Business congratulation page

### PR DESCRIPTION
**Pull request by rcanepa**
#### Changes proposed in this Pull Request

* Include a card for Backup and Scan in the congratulation page we show when users upgrade to Business plan.

#### Testing instructions

You can test this PR by going directly to the URL of the congratulation page (as the instructions below indicate), or you can simulate the whole purchasing process. If you want to do the latter, you'll need a WPCOM site that can be upgraded to a Business plan.

* **Prerequisite**: WPCOM site with Free, Personal, or Premium plan.
* Run this PR (Calypso).
* Visit `http://calypso.localhost:3000/plans/<your-site-slug>?customerType=business`.
* Upgrade your site to Business.
* (You should see the Congratulations page.)
* Verify that the Backup card is not present.
* Verify that the Scan card is not present.
* Add to the URL the `flags=jetpack/features-section` query parameter to activate the Jetpack section.
* Reload the page.
* Verify that the Backup card is present and its link points to the new Backup section.
* Verify that the Scan card is present and its link points to the new Scan section.

Fixes 1179060693083348-as-1179702911174266

##### `jetpack/features-section` flag enabled.
image

##### `jetpack/features-section` flag disabled.
image

### CODE REVIEW

File: client/my-sites/checkout/checkout-thank-you/business-plan-details.jsx [hunk 1]
```diff
@@ -49,6 +51,32 @@ const BusinessPlanDetails = ( {
...
+						buttonText={ i18n.translate( 'See the latest backup' ) }
+						href={ `/backup/${ selectedSite.slug }` }
+						onClick={ trackOnboardingButtonClick }
+					/>
+
+					<PurchaseDetail
+						icon={ <img alt="" src={ jetpackScanImage } /> }
+						title={ i18n.translate( 'Scan your site' ) }
+						description={ i18n.translate(
+							'Scan gives you automated scanning and one-click fixes to keep your site ahead of security threats.'
```
Comment (a8ci18n) [inline client/my-sites/checkout/checkout-thank-you/business-plan-details.jsx:71]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **17** times:
`translate( 'Jetpack Scan uses automated scanning and one-click fixes to keep your site ahead of security threats.' )` *ES Score: 9*
See 1 additional suggestions in the [PR translation status page](https://translate.wordpress.com/localci/status/automattic/wp-calypso/add/backup-scan-business-congratulation)

**Comment by cleacos** [comment]
I cannot test it with that receipt Id:

image

Another way to test it?

**Comment by rcanepa** [comment]
> I cannot test it with that receipt Id:
> 
> image
> 
> Another way to test it?

😞 
I guess the only way left to test this would be to create a WPCOM site with a Free, Personal, or Premium plan and then upgrade it to a Business one. I'll update the instructions to reflect that.

**Comment by cleacos** [comment]
Branch rebased. Conflict with `client/assets/images/illustrations/jetpack-scan.svg`

File: client/my-sites/checkout/checkout-thank-you/business-plan-details.jsx [hunk 2]
```diff
@@ -49,6 +51,32 @@ const BusinessPlanDetails = ( {
 		<div>
 			{ googleAppsWasPurchased && <GoogleAppsDetails isRequired /> }
 
+			{ isEnabled( 'jetpack/features-section' ) && (
+				<>
+					<PurchaseDetail
+						icon={ <img alt="" src={ jetpackBackupImage } /> }
+						title={ i18n.translate( 'Check your backups' ) }
+						description={ i18n.translate(
+							'Backup gives you granular control over your site, with the ability to restore it to any previous state, and export it at any time.'
```
Comment (a8ci18n) [inline client/my-sites/checkout/checkout-thank-you/business-plan-details.jsx:60]: :information_source: String reuse speeds up translation and improves consistency. The following string might make a good alternative and has already been translated **7** times:
`translate( 'Granular control over your site, with the ability to restore it to any previous state, and export it at any time.' )` *ES Score: 8*

**Comment by a8ci18n** [comment]
This Pull Request is now available for translation here: https://translate.wordpress.com/deliverables/3847254

Thank you @rcanepa for including a screenshot in the description! This is really helpful for our translators.

**Comment by a8ci18n** [comment]
Translation for this Pull Request [has now been finished](https://translate.wordpress.com/deliverables/overview/3847254).

**Pull request merged**